NitroxServer-Subnautica/Program.cs

```diff
@@ -24,13 +25,19 @@ private static void Main(string[] args)
             NitroxServiceLocator.InitializeDependencyContainer(new SubnauticaServerAutoFacRegistrar());
             NitroxServiceLocator.BeginNewLifetimeScope();
 
-            Server server;
-
+            Server server = null;
             try
             {
                 server = NitroxServiceLocator.LocateService<Server>();
+                Log.Info($"Loaded save\n{server.SaveSummary}");
                 server.Start();
             }
+            catch (SocketException e)
+            {
+                Log.Error("Unable to start server", e);
+                server?.Stop();
```

Comment:
LiteNetLib actually eats the exception so I changed it again to just abort and log.

Reply:
I don't think we want to crash on any error since most of the time it will just fail to save/persist something which isn't bad enough to crash the whole thing imo.